book/page_creation.rst

```diff
@@ -225,595 +215,282 @@ the new route that defines the URL of the page that you're about to create:
-                <title>{% block title %}Welcome!{% endblock %}</title>
-                {% block stylesheets %}{% endblock %}
-                <link rel="shortcut icon" href="{{ asset('favicon.ico') }}" />
-            </head>
-            <body>
-                {% block body %}{% endblock %}
-                {% block javascripts %}{% endblock %}
-            </body>
-        </html>
-
-    .. code-block:: html+php
-
-        <!-- app/Resources/views/base.html.php -->
-        <!DOCTYPE html>
-        <html>
-            <head>
-                <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-                <title><?php $view['slots']->output('title', 'Welcome!') ?></title>
-                <?php $view['slots']->output('stylesheets') ?>
-                <link rel="shortcut icon"
-                    href="<?php echo $view['assets']->getUrl('favicon.ico') ?>" />
-            </head>
-            <body>
-                <?php $view['slots']->output('_content') ?>
-                <?php $view['slots']->output('javascripts') ?>
-            </body>
-        </html>
-
-The base template file defines the HTML layout and renders the ``body`` block
-that you defined in the ``index.html.twig`` template. It also renders a ``title``
-block, which you could choose to define in the ``index.html.twig`` template.
-Since you did not define the ``title`` block in the child template, it defaults
-to "Welcome!".
-
-Templates are a powerful way to render and organize the content for your
-page. A template can render anything, from HTML markup, to CSS code, or anything
-else that the controller may need to return.
-
-In the lifecycle of handling a request, the templating engine is simply
-an optional tool. Recall that the goal of each controller is to return a
-``Response`` object. Templates are a powerful, but optional, tool for creating
-the content for that ``Response`` object.
-
-.. index::
-   single: Directory Structure
-
-The Directory Structure
------------------------
-
-After just a few short sections, you already understand the philosophy behind
-creating and rendering pages in Symfony. You've also already begun to see
-how Symfony projects are structured and organized. By the end of this section,
-you'll know where to find and put different types of files and why.
-
-Though entirely flexible, by default, each Symfony :term:`application` has
-the same basic and recommended directory structure:
-
-``app/``
-    This directory contains the application configuration.
-
-``src/``
-    All the project PHP code is stored under this directory.
-
-``vendor/``
-    Any vendor libraries are placed here by convention.
-
-``web/``
-    This is the web root directory and contains any publicly accessible files.
-
-.. seealso::
-
-    You can easily override the default directory structure. See
-    :doc:`/cookbook/configuration/override_dir_structure` for more
-    information.
-
-.. _the-web-directory:
```

Comment:
There seem to be references in other documents to this section (see the Travis build).